rollup.cssBundler.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs')
```

Comment:
> I just also kept the custom rollup plugin because it meant we could add comments for each section of CSS (in the resulting file) saying which transform it originated from.

^ not sure that's as useful as I thought it was at the time though... :)